tests/mock_tables/counters_db.json

```diff
@@ -203,108 +203,6 @@
     "SAI_PORT_STAT_PFC_7_RX_PKTS": "8",
     "SAI_PORT_STAT_PFC_7_TX_PKTS": "8"
   },
-  "COUNTERS:oid:0x1000000000020": {
```

Comment:
Is this removed as it is not used?

Reply:
It is not used.
But the intention is to corrupt the Redis and make sure snmpagent could tolerate inconsistent data in COUNTERS_DB.